### PR TITLE
[MISC] Fix handling of azure storage relation databag

### PR DESCRIPTION
**Pull request by welpaolo**
This PR address an issue of Azure Storage databag is empty due to missing configuration of azure-storage-integrator and the charm was crashing.

### CODE REVIEW

File: src/core/context.py [hunk 1]
```diff
@@ -20,6 +20,13 @@
 OATHKEEPER = "auth-proxy"
 AUTHORIZED_USERS = "authorized-users"
 AUTH_PROXY_HEADERS = ["X-User", "X-Email"]
+AZURE_MANDATORY_OPTIONS = [
+    "secret-key",
+    "secret-key",
```
Comment (theoctober19th) [inline src/core/context.py:25]: duplicate: secret-key

Comment (theoctober19th) [reply src/core/context.py:25]: missing: access-key

**Review by theoctober19th — COMMENTED** [review]
There's a small typo. Otherwise, looks solid.

**Review by theoctober19th — APPROVED** [review]

File: src/core/context.py (hunk 1 above)
Comment (welpaolo) [reply src/core/context.py]: fixed access-key is not in the databag

**Pull request merged**